lib/fluent/supervisor.rb

```diff
@@ -158,39 +242,11 @@ def start
       set_system_config
 
       dry_run if @dry_run
-      start_daemonize if @daemonize
       setup_rpc_server if @rpc_endpoint
```

Comment:
Following 3 instructions should be in one `if` block.
```
if @rpc_endpoint
  setup_rpc_server
  setup_rpc_get_dump if @enable_get_dump
  run_rpc_server
end
```
Or, these 3 methods might be just one method (if it looks better than 3 methods).